polkadot/runtime/rococo/src/lib.rs

```diff
@@ -1714,6 +1714,7 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 
+		#[api_version(99)]
```

Comment:
This is a problem. We can not have a release with version 99. We have to pick a proper version, which has consequences (which should be fine): We have to release this API next in production. We can then no longer release any other new functionality without also releasing async backing.

Reply:
To be more correct: We have to determine now what the next version is exactly. By making it no longer 99, this version is stable and must no longer be changed/modified in any way without bumping the version again.

Reply:
What we need is a way to stabilize the API without automatically upgrading all runtimes to it